Code/GraphMol/RGroupDecomposition/RGroupCore.h

```diff
@@ -50,15 +50,17 @@ struct RCore {
 
   // Return a copy of core where dummy atoms are replaced by
   // the respective matching atom in mol, while other atoms have
-  // their aromatic flag and formal charge copied from 
+  // their aromatic flag and formal charge copied from
   // the respective matching atom in mol
   ROMOL_SPTR replaceCoreAtomsWithMolMatches(bool &hasCoreDummies,
                                             const ROMol &mol,
                                             const MatchVectType &match) const;
 
-  // Final core returned to user with dummy atoms and bonds set to those in the
-  // match
-  RWMOL_SPTR coreWithMatches(const ROMol &coreReplacedAtoms) const;
+  // Final core returned to user, created by extracting core from target
+  // molecule
+  RWMOL_SPTR extractCoreFromMolMatch(
+      bool &hasCoreDummies, const ROMol &mol, const MatchVectType &match,
+      const RGroupDecompositionParameters &params) const;
```

Comment:
I commented elsewhere on this, but I think it makes more sense to declare this as:
```suggestion
  std::pair<RWMOL_SPTR,bool> extractCoreFromMolMatch(
      const ROMol &mol, const MatchVectType &match,
      const RGroupDecompositionParameters &params) const;
```

Reply:
> I commented elsewhere on this, but I think it makes more sense to declare this as:

I've made that change